feat(input): add s size for filled view

Heymdall opened the pull request:
WIP Добавлен s размер для filled view у select, input, textarea

## Мотивация и контекст
Размер нужен в одном из новых проектов. 
Еще нужна верификация от дизайнеров, просто первая итерация

SiebenSieben commented:
Кажется, великоват отступ между лейблом и текстом в Textarea:
<img width="386" alt="Screenshot 2019-10-09 at 18 54 04" src="https://user-images.githubusercontent.com/109410/66498408-abe75700-eac6-11e9-9fb9-011c475108f8.png">

У размера m отступы Textarea соответствуют инпуту.

SiebenSieben commented:
Если есть leftAddons, то лейбл получился левее текста:
<img width="210" alt="Screenshot 2019-10-09 at 18 54 43" src="https://user-images.githubusercontent.com/109410/66498494-d802d800-eac6-11e9-8dcf-2168ca29ab19.png">

Heymdall commented:
> Кажется, великоват отступ между лейблом и текстом в Textarea:
> <img alt="Screenshot 2019-10-09 at 18 54 04" width="386" src="https://user-images.githubusercontent.com/109410/66498408-abe75700-eac6-11e9-9fb9-011c475108f8.png">
> 
> У размера m отступы Textarea соответствуют инпуту.

fixed

> Если есть leftAddons, то лейбл получился левее текста:
> <img alt="Screenshot 2019-10-09 at 18 54 43" width="210" src="https://user-images.githubusercontent.com/109410/66498494-d802d800-eac6-11e9-8dcf-2168ca29ab19.png">

fixed
